app/src/main/res/layout-land/walkthrough_final_fragment.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.walkthrough.end.WalkthroughFinalViewModel" />
+    <variable
+      name="presenter"
+      type="org.oppia.app.walkthrough.end.WalkthroughFinalFragmentPresenter" />
+  </data>
```

Comment:
Formatting is incorrect. You can use https://docs.google.com/document/d/1AHIJagMfATgpbNCFmYoYaSqzsKp8JjTgPLQFXEFJdUA/edit?usp=sharing to fix that formatting.

Reply:
Thanks for this